data-raw/instream/san_joaquin/san_joaquin_river.Rmd

```diff
@@ -15,22 +15,25 @@ library(scales)
 
 ```
 ## Future Data Improvements
-[Future data improvements bullets]
+None Planned 
 
 ## Instream Spawning and Rearing Habiat
 
-**Data Source:** 
-[add data source]
+**Data Source:**     
+[Final Lower Tuolumne Instream Flow Study Report](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/Stillwater2010_Tuolumne_P-2299-075_71_DP_FLA_AttC_StudyRept_IFIM_AppA-H_140428.pdf) (pg. 108-109)    
+[Stanislaus Aceituno 1993](https://cvpiahabitat-r-package.s3-us-west-2.amazonaws.com/cvpia-sit-model-inputs/Aceituno1993_StanislausRiverIFIMStudy.pdf) (pg 19)    
+[Stanislaus FWS 2012](https://cvpiahabitat-r-package.s3-us-west-2.amazonaws.com/cvpia-sit-model-inputs/CVPIA_Annual_Progress_Report_Fiscal_Year_2012.pdf) (pg 6)    
+[Merced ID 2013](https://cvpiahabitat-r-package.s3-us-west-2.amazonaws.com/cvpia-sit-model-inputs/MID2013_MercedIFIMStudy.pdf) (pg 67-75)    
 
-Instream spawning and rearing habitat for Fall Run Chinook Salmon in the San Joaquin River is based on data from [data source description]. The [DWR FERC instream spawning and rearing habitat data] was provided by [Mark Gard](mailto:mark_gard@fws.gov) from the U.S. Fish and Wildlife Service in a spreadsheet (see '~/cvpiaHabitat/data-raw/mark_gard_data/IFIMWUA.xlsx'). 
+No watershed specific salmonid habitat data was available for the San Joaquin River. A regional weighted usable area (WUA) relationship with flow was derived for the San Joaquin River by averaging the WUA values on the three tributaries of the San Joaquin. Spawning and rearing data from the Merced, Stanislaus, and Tuolumne Rivers was used to estimate spawning and rearing WUA for the San Joaquin. The data for these three tributaries was provided by [Mark Gard](mailto:mark_gard@fws.gov) from the U.S. Fish and Wildlife Service in a spreadsheet (see '~/cvpiaHabitat/data-raw/mark_gard_data/IFIMWUA.xlsx'). Mark Gard instructed us to use A18:E48 of the 'Merced' tab, I4:N60 of the 'Stanislaus' tab, and A2:H32 in the 'Tuolumne' tab within his 'IFIMWUA.xlsx' spreadsheet. 
```

Comment:
Change was to were in: 'Spawning and rearing data from the Merced, Stanislaus, and Tuolumne Rivers **was** used to estimate spawning and rearing WUA for the San Joaquin.'

Reply:
Fixed Grammar.